code/src/components/app-table/components/app-table-box/AppTableBox.test.tsx

```diff
@@ -0,0 +1,18 @@
+import { render } from "@testing-library/react";
+
+import AppTableBox from "@/components/app-table/components/app-table-box/AppTableBox";
+
+describe("AppTableBox", () => {
+  test("renders with children", () => {
+    const { getByText } = render(
```

Comment:
why `render` instead `screen` ? Look at our approaches
Also in other cases. don't write the same. check it in other places